.github/CONTRIBUTING.md

```diff
@@ -1,34 +1,90 @@
 # Contributing to Kitura
 
-We welcome contributions, and request you follow these guidelines.
+This document contains information and guidelines about making contributions. For help, contact us on [Slack](http://swift-at-ibm-slack.mybluemix.net/).
 
- - [Raising issues](#raising-issues)
+We love contributions to the Kitura project, but request you follow the guidelines below.
```

Comment:
I think it'd be good to combine the sentences in lines 3 and 5 into one chunk of text.  Perhaps something like `We love contributions to the Kitura project and request you follow the guidelines below.  If you have any questions, or need any help, ask us on [Slack]`

Reply:
✅